### ISSUE TITLE
Reopening, as I'm not seeing consistent id selection:

### ISSUE THREAD

**Issue by sajManzoor** [facebook/react-native#22524]
```
<TouchableWithoutFeedback accessible={false}>
  <View testId='foo'></View>
</TouchableWithoutFeedback>

```

In this case the accessibilityID 'foo' cannot be found.

_Originally posted by @kidmillions in https://github.com/facebook/react-native/issues/11869#issuecomment-272976331_
